theano/tensor/tests/test_basic.py

```diff
@@ -1824,14 +1826,45 @@ def test_py_c_match():
     inplace=True,
     skip=skip_scipy)
 
+# like chi2sf, polygamma takes two inputs, a value (x) and the order (k).  
+# also not sure how to deal with that here...   
+_good_broadcast_unary_polygamma = dict(
+    normal=(rand_ranged(1, 10, (2, 3)),),
+    empty=(numpy.asarray([], dtype=config.floatX),),)
+_grad_broadcast_unary_polygamma = dict(
```

Comment:
To handle the 2 inputs, look at the SubTester in this file. You just need to add a second value to tuples with the 2nd value.